cpp/source/supercapacitor.templates.h

```diff
@@ -444,70 +444,4 @@ void SuperCapacitor<dim>::get_voltage(double &voltage) const
 {
   (*this->post_processor).get("voltage", voltage);
 }
-
-template <int dim>
-void SuperCapacitor<dim>::print_data(std::ostream &os) const
-{
-  double current;
-  double voltage;
-  (*this->post_processor).get("voltage", voltage);
-  (*this->post_processor).get("current", current);
-  os << boost::format("  %10.5f  %10.7f  \n") % current % voltage;
-
-  static int i                  = 0;
```

Comment:
Unless this is exactly what we want, I think that we should talk about what we want exactly. This is the function for the old supercapacitor. There is nothing similar for the new supercapacitor, so it may be useless to save this code.